osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1670,6 +1670,35 @@
         "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/hash.svg"
       },
       "coingecko_id": "provenance-blockchain"
+    },
+    {
+      "description": "GLX is the staking token of the Galaxy Chain",
```

Comment:
Which asset description is correct? If this, can the chain registry be corrected?
https://github.com/cosmos/chain-registry/blob/master/galaxy/assetlist.json

Reply:
Yes chain registry more corrected

Reply:
yep, this new description looks good. Please propose correction on Chain Registry

Reply:
I push a PR to correct chain registry. I will keep you update